x/pricefeed/genesis.go

```diff
@@ -5,23 +5,23 @@ import (
 )
 
 // InitGenesis sets distribution information for genesis.
-func InitGenesis(ctx sdk.Context, keeper Keeper, data GenesisState) {
-
-	// Set the assets and oracles from params
-	keeper.SetParams(ctx, data.Params)
+func InitGenesis(ctx sdk.Context, keeper Keeper, gs GenesisState) {
```

Comment:
nit: `InitGenesis` could call `data.Validate` to match other modules

Reply:
0069718